vm/vm_test.go

```diff
@@ -74,6 +74,22 @@ foo
 	}
 }
 
+func initTestVM() *VM {
```

Comment:
I separate vm's initialization from `testEval` because in this way we can access the vm's internal state if we want to.